x/dex/keeper/msg_server.go

```diff
@@ -119,6 +119,8 @@ func (k MsgServer) PlaceLimitOrder(
 		return &types.MsgPlaceLimitOrderResponse{}, err
 	}
 
+	totalLimitOrders()
```

Comment:
Not sure what this metric does either.

Reply:
We only want to track active limit orders. I think this can be achieved with the same strategy used for the tick liquidity tracking